src/utils/suggest.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"sort"
+
+	"github.com/texttheater/golang-levenshtein/levenshtein"
+)
+
+// Suggest implements levenshtein-based suggestions on a sequence of items.
+func Suggest(needle string, haystack []string, maxSuggestionDistance int) []string {
+	r := []rune(needle)
+	options := make(suggestions, 0, len(haystack))
+	for _, straw := range haystack {
+		distance := levenshtein.DistanceForStrings(r, []rune(straw), levenshtein.DefaultOptions)
+		if distance <= maxSuggestionDistance {
+			options = append(options, suggestion{s: straw, dist: distance})
+		}
+	}
+	sort.Sort(options)
+	ret := make([]string, len(options))
+	for i, o := range options {
+		ret[i] = o.s
+	}
+	return ret
+}
+
+// Suggest implements levenshtein-based suggestions on a sequence of items and
+// produces a single message from them.
+func SuggestMessage(needle string, haystack []string, maxSuggestionDistance int) string {
```

Comment:
PrettyPrintSuggestion or smth like that? Had to read through the code of both functions to understand the difference.